frontend/src/components/app/app.tsx

```diff
@@ -97,6 +98,10 @@ const App: FC = () => {
           path={AppRoute.PROFILE}
           element={<AuthorizedProtectedRoute component={<UserDetails />} />}
         />
+        <Route
+          path={AppRoute.MY_COURSES}
+          element={<AuthorizedProtectedRoute component={<MyCourses />} />}
```

Comment:
```suggestion
          element={<AuthorizedWrapper><MyCourses /></AuthorizedWrapper>}
```
Since the page is not protected by any permissions